tests/console/docker_image.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LINUX GmbH
```

Comment:
s/SUSE LINUX GmbH/SUSE LLC/

Reply:
It's never been SUSE LLC AFAIK, same as in the .spec file headers.
Is there a specific reason it's different here?

Reply:
yes, because @coolo said so

Reply:
Ok, I'll change it. I'm curious though: Why?

Reply:
I guess because "SUSE LLC > SUSE Linux GmbH" and because it's the correct legal entity. Also I think it's "SUSE Linux" and not "SUSE LINUX".

Reply:
According to suse.com, only "SUSE LINUX GmbH" and "SUSE PLC" exist, I can't find any LLC anywhere

Reply:
They have now an even funnier name :)
"SUSE" and the SUSE logo are trademarks of SUSE IP Development Limited or its subsidiaries or affiliates

Reply:
Argh. What's the correct one now?

Reply:
use SUSE LLC in this repo - we might mass convert later :)

Reply:
Ok, then everything should be ok already.